src/app.js

```diff
@@ -43,7 +43,7 @@ app.use(passport.initialize());
 app.use(passport.session());
 
 
-const port = process.env.PORT || 4000;
+const port = process.env.PORT || 3300;
```

Comment:
This change is unnecessary since you have a `PORT` variable you can use to change port.